src/App.js

```diff
@@ -1,26 +1,28 @@
+  
 import React, { useState } from 'react';
 import './App.css';
-import { Link, Switch, Route, Redirect } from 'react-router-dom';
+import {Switch, Route,} from 'react-router-dom';
 import FirebaseTesting from './pages/FirebaseTest';
 import FooterTabs from './components/FooterTabs';
 import ShoppingList from './pages/ShoppingList';
 import AddItem from './pages/AddItem';
-import NewListButton from './components/NewListButton';
+/*import NewListButton from './components/NewListButton';*/
 import * as ls from 'local-storage';
+import Home from './pages/Home';
 
 function App() {
-	const [token, setToken] = useState(ls.get('shoppingListToken'));
+	const [token] = useState(ls.get('shoppingListToken'));
```

Comment:
```suggestion
	const [token, setToken] = useState(ls.get('shoppingListToken'));
```
We need to also declare setToken, since we need access to mutate token state in child components of App.